### PR TITLE
Clarify pan/tilt/zoom constraints (#225)

### PR DESCRIPTION
**Pull request by eehakkin**
Hopefully this clarifies the spec and resolves the concerns raised in #225.

There are no functional or algorithmic changes as that is the consensus reached during discussion in #225 as I understood.

### CODE REVIEW

**Comment by beaufortfrancois** [comment]
@guidou Does that look good to you as well?

File: index.bs [hunk 1]
```diff
@@ -776,19 +776,26 @@ When the {{getSettings()}} method is invoked on a video stream track, the user a
...
+    Any algorithm which uses a {{MediaTrackConstraintSet}} object and its {{MediaTrackConstraintSet/pan}} dictionary member which exists after a possible normalization MUST <a>request permission to use</a> (as defined in [[!permissions]]) a PermissionDescriptor with its name member set to "camera" and its {{CameraDevicePermissionDescriptor/panTiltZoom}} member set to true, and, optionally, consider its deviceId member set to any appropriate device's deviceId.
+
+    <div class="note">
+      The {{MediaTrackConstraintSet/pan}}, {{MediaTrackConstraintSet/tilt}} and {{MediaTrackConstraintSet/zoom}} dictionary members exists after a possible normalization if the normalized value is a double value or a {{ConstrainDoubleRange}} value (whether empty or not).
+      A boolean value of true is normalized to an empty {{ConstrainDoubleRange}} value thus the dictionary member exists after a normalization.
+      A boolean value of false is normalized to no value thus the dictionary member does not exist after a normalization.
+      An empty {{ConstrainDoubleRange}} value implies no constraints but only a permission request.
```
Comment (guidou) [inline index.bs:785]: I would say it implies a permission and capability request.

Comment (eehakkin) [reply index.bs:785]: Done.

**Comment by guidou** [comment]
lgtm, but please add @alvestrand as reviewer too

**Comment by riju** [comment]
@alvestrand : Harald, PTAL.

**Pull request merged**